packages/trader/src/App/Components/Elements/Modals/ServicesErrorModal/__tests__/services-error-modal.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
Please, pin the image of the test coverage report to you card> Example: https://redmine.deriv.cloud/issues/90801#Kate/_To_migrate_turbos-info,_payout-per-point_to_TS_together_with_their_tests
More over, you can find a lot of useful information like how many branches/functions were tested and etc

Reply:
Working on it :)